optuna/integration/mlflow.py

```diff
@@ -132,8 +132,8 @@ def __init__(
         self,
         tracking_uri: Optional[str] = None,
         metric_name: Union[str, Sequence[str]] = "value",
-        nest_trials: bool = False,
-        tag_study_user_attrs: bool = False,
```

Comment:
After reading #1918 I'm proposing to remove this argument and log study user attributes by default. Reasons:
* Trial user attributes are always logged anyway. This introduces discrepancies in behavior.
* User must explicitly set study attributes via `study.set_user_attr` anyway. In my opinion this reduces chance for accidentally cluttering mlflow experiment.
* After manually setting attributes, it's reasonable for user to expect that they will be logged, without additional "are you sure?" question to answer in callback.
* Complicates API.
* Wasn't really tested.

Alternatively, we can make both trial and study attributes optional, to at least unify behavior. Let me know what you guys think.

Reply:
Thank you for your proposal. it makes sense. Let unify the logging *`.user_attrs` by removing the argument.